Documentation/botr/readytorun-format.md

```diff
@@ -427,6 +496,27 @@ The module override index translation algorithm is as follows (**ILAR** = *the n
 
 **TODO**: document attribute presence encoding
 
+## READYTORUN_SECTION_ASSEMBLIES (v3.2+)
+
+This section is only present in composite R2R files. It is a straight binary array of the entries `READYTORUN_SECTION_ASSEMBLIES_ENTRY` parallel to the indices in the manifest metadata AssemblyRef table in the sense that it's a linear table where the row indices correspond to the equivalent AssemblyRef indices. Just like in the AssemblyRef ECMA 335 table, the indexing is 1-based (the first entry in the table corresponds to index 1).
+
+```C++
+struct READYTORUN_SECTION_ASSEMBLIES_ENTRY
+{
+    DWORD  CorHeaderRva;          // RVA of the input MSIL metadata COR header
+    DWORD  AvailableTypesRva;     // RVA of the available types table
+    DWORD  AvailableTypesSize;    // Size in bytes of the available types table
+    DWORD  MethodEntrypointsRva;  // RVA of the method entrypoint table
+    DWORD  MethodEntrypointsSize; // Size of the method entrypoint table
+};
+```
+
+**TODO:** I suspect that `READYTORUN_SECTION_DEBUG_INFO`, `READYTORUN_SECTION_METHODCALL_THUNKS` and / or `READYTORUN_SECTION_INLINING_INFO` may also require changes specific to the composite R2R file format.
```

Comment:
`READYTORUN_SECTION_DEBUG_INFO` is functionally okay to just merge all the R2R code debug info into a single table mapping back to IL offsets / locals since it maps from native offset to offset within IL bodies, variable numbers. The debug engine will need to do the work of making sense of the merged assembly.

Reply:
I have removed READYTORUN_SECTION_DEBUG_INFO from the 'suspect list' per your suggestion in 8th commit.